app/routes/admin/activities/remove.js

```diff
@@ -0,0 +1,81 @@
+const { routes, views } = require('../../../constants/admin')
+const { sources, keys } = require('../../../constants/cdo/activity')
+const { admin } = require('../../../auth/permissions')
+const ViewModel = require('../../../models/common/confim')
+const { getActivityById } = require('../../../api/ddi-index-api/activities')
+const { validatePayload } = require('../../../schema/portal/common/confirm')
+const { deleteActivity } = require('../../../api/ddi-index-api/activities')
+const { ActivityRemovedViewModel } = require('../../../models/admin/activities/builder')
+const { getUser } = require('../../../auth')
+const { notFoundSchema } = require('../../../schema/portal/common/single-remove')
+
+const backLink = routes.activities.index.get
+
+const getActivityConstants = activity => ({
+  recordTypeText: 'activity',
+  recordType: 'activity',
+  action: 'remove',
+  buttonText: 'Remove activity',
+  confirmReferenceText: activity.label,
+  confirmHint: `${activity.activity_source?.name === sources.dog ? 'Dog' : 'Owner'} record: something we ${activity.activity_type?.name === keys.sent ? 'send' : 'receive'}`
+})
```

Comment:
nice touch this, just a thought though, are we using the activity constants from the constants file anymore?

Reply:
I can still reference the constants, as a hybrid